src/bp-templates/bp-nouveau/buddypress/notifications/on-screen.php

```diff
@@ -1,4 +1,4 @@
-<?php if ( bp_has_notifications( bp_ajax_querystring( 'notifications' ) . '&user_id=' . get_current_user_id() . '&is_new=1' ) ) : ?>
+<?php if ( bp_has_notifications( bp_ajax_querystring( 'notifications' ) . '&user_id=' . get_current_user_id() . '&is_new=1&per_page=500' ) ) : ?>
```

Comment:
@asaquzzamanbb Why we are loading the 500 notifications at a time?

Reply:
this is not the part of this PR. Its a mistake. 
Removed here https://github.com/buddyboss/buddyboss-platform/pull/2290/commits/8b0cf52e3e6ab3d1b53634e3166abc67001a14d2